config/environments/development.rb

```diff
@@ -50,4 +50,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  config.force_ssl = Settings.use_ssl
```

Comment:
Why do we want to force ssl/https for local development?

Reply:
It configurable, via `Settings`.
Also being compliant with `https` is a requirement to use Dfe Signin.